mirdata/beatles.py

```diff
@@ -22,46 +22,22 @@
 )
 
 
-def download(data_home=None, force_overwrite=False):
+def download(data_home=None):
     save_path = utils.get_save_path(data_home)
-    dataset_path = os.path.join(save_path, BEATLES_DIR)
 
-    if force_overwrite:
-        utils.force_overwrite_all(BEATLES_ANNOT_REMOTE, dataset_path, data_home)
-
-    if utils.check_validated(dataset_path):
-        print(
-            """
-                The {} dataset has already been downloaded and validated.
-                Skipping download of dataset. If you feel this is a mistake please
-                rerun and set force_overwrite to true
-                """.format(
-                BEATLES_DIR
-            )
+    print(
+        """
+        Unfortunately the audio files of the Beatles dataset are not available
+        for download. If you have the Beatles dataset, place the contents into
+        a folder called {beatles_dir} with the following structure:
+            > {beatles_dir}/
+                > annotations/
+                > audio/
+        and copy the {beatles_dir} folder to {save_path}
+    """.format(
+            beatles_dir=BEATLES_DIR, save_path=save_path
         )
-        return
-
-    download_path = utils.download_from_remote(
```

Comment:
this should stay in - it's downloading the annotations from the beatles' dataset website